docs/diagrams/DeleteSequenceDiagram.puml

```diff
@@ -33,16 +32,16 @@ create DeleteCommand
 DeleteCommandParser -> DeleteCommand
 activate DeleteCommand
 
-DeleteCommand --> DeleteCommandParser : d
+DeleteCommand --> DeleteCommandParser
 deactivate DeleteCommand
 
-DeleteCommandParser --> AddressBookParser : d
+DeleteCommandParser --> AddressBookParser
 deactivate DeleteCommandParser
 'Hidden arrow to position the destroy marker below the end of the activation bar.
 DeleteCommandParser -[hidden]-> AddressBookParser
 destroy DeleteCommandParser
 
-AddressBookParser --> LogicManager : d
+AddressBookParser --> LogicManager : command
 deactivate AddressBookParser
```

Comment:
this is the thing I'm referring to. Over here you use the label "command", but it's not 100% clear where that came from, which is why I think you should replace all the d's you removed with this "command" label